main.js

```diff
@@ -10,3 +12,142 @@ function toggle() {
+                            </ul>
+                            <a class="button-interact popping${i}">See Project</a>
+                        </div>
+                      </div>
+                    `;
+  grid.appendChild(div);
+}
+
+// Popup-window cards
+
+for (let i = 0; i < obj.length; i += 1) {
+  const xxx = `.popping${i}`;
+  const open = document.querySelector(xxx);
+  open.addEventListener('click', () => {
+    const main = document.createElement('div');
+    main.className = 'main';
+    const popup = document.createElement('div');
+    popup.className = 'popup';
+    popup.innerHTML = `
+                            <div class="work-grid2">
+                                <div class="container3 spacing1">
+                                    <div class="item2-pop flex-column">
+                                        <button class="btn close" type="button"></button>
+                                        <img class="snapshoot2" src="${obj[i].image}" alt="portfolio project image ${i + 1}">
+                                    </div>
+                                    <div class="item-pop block-column">
+                                        <div class="wp-div">
+                                            <button class="button-interact antispacing1 display1"><a class="deco" href="${obj[i].live}" target="_blank">See Live <i
+                                                        class="i-back1"></i></a></button>
+                                            <button class="button-interact antispacing1 display1"><a class="deco" href="${obj[i].source}" target="_blank">See Source <i
+                                                        class="i-back2"></i></a></button>
+                                        </div>
+                                        <h3 class="h3">${obj[i].name}</h3>
+                                        <ul class="skills-wp">
+                                            <li class="skills1">${obj[i].technologies[0]}</li>
+                                            <li class="skills1">${obj[i].technologies[1]}</li>
+                                            <li class="skills1">${obj[i].technologies[2]}</li>
+                                            <li class="skills1">${obj[i].technologies[3]}</li>
+                                        </ul>
+                                        <p class="p3">${obj[i].description}</p>
+                                        <button class="button-interact antispacing1 display2"><a class="deco" href="${obj[i].live}" target="_blank">See Live <i
+                                                    class="i-back1"></i></a></button>
+                                        <button class="button-interact antispacing1 display2"><a class="deco" href="${obj[i].source}" target="_blank">See Source <i
+                                                    class="i-back2"></i></a></button>
+                                    </div>
```

Comment:
- [ ] Great job on implementing the modal window 👏 kindly try to follow the design as much as possible:
- [ ] For the desktop version, the image should be bigger than it is right now, and the window is much bigger than the screen (I couldn't see the close button)

## Your design:
![image](https://user-images.githubusercontent.com/50721479/202765067-b532c86b-3822-45d0-9588-dfe091e3cd11.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/202764908-b07b402b-a0ea-41a5-9833-74d64d6f17e3.png)